CMakeLists.txt

```diff
@@ -154,13 +154,12 @@ find_package(Python3 COMPONENTS Interpreter)
 configure_file(config.h.in config.h)
 
 if(WIN32)
-    install(FILES COPYING DESTINATION docs RENAME COPYING.txt)
-    install(DIRECTORY examples/ DESTINATION docs/examples)
+    install(FILES COPYING DESTINATION . RENAME COPYING.txt)
     if(FluidSynth_FOUND)
         install(DIRECTORY soundfonts/ DESTINATION soundfonts)
     endif()
 else()
-    install(FILES COPYING DESTINATION "share/doc/${PROJECT_SHORTNAME}")
+    install(FILES COPYING DESTINATION "share/${PROJECT_SHORTNAME}")
```

Comment:
This change is wrong. Documentation is supposed to go to a subdirectory of `/usr/share/doc`.